.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: "Mark Stale Issues and PRs"
+
+on:
+  schedule:
+    - cron: '0 */8 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-pr-stale: 30
+        days-before-issue-stale: 30
+        stale-issue-message: 'This issue has not seen activity for a while. It will be closed in 30 days unless further activity is detected'
+        stale-pr-message: 'This PR has not seen activitiy for a while. It will be closed in 30 days unless further activity is detected.'
+        close-issue-message: 'This issue has been closed because of inactivity.'
+        close-pr-message: 'This PR has been closed because of inactivity.'
+        exempt-pr-labels: 'WIP, on-hold, needs review'
```

Comment:
Was about to write a whole thing but then I saw this, does the same exist for issues?

Reply:
This is both for issues and PRs, tho I haven't written explicit conditions for issues here. Do we need it for issues, if so, will update the workflow :)

Reply:
Yep, I'd use the same labels + `roadmap`